src/d/d_gameover.cpp

```diff
@@ -348,15 +371,47 @@ BOOL dDlst_GameOverScrnDraw_c::anime1(int idx) {
 }
 
 /* 8018F334-8018F4A8       .text anime2__24dDlst_GameOverScrnDraw_cFi */
-void dDlst_GameOverScrnDraw_c::anime2(int) {
-    /* Nonmatching */
+BOOL dDlst_GameOverScrnDraw_c::anime2(int idx) {
+    static const s16 rot[6] = {
+        40, 30, 20,
+        -20, -30, -40
+    };
+
+    BOOL ret = FALSE;
+
+    if (letter[idx].mUserArea < 7) {
+        if (letter[idx].mUserArea == 0) {
+            s32 rand = cM_rndF(6.0f);
+
+            if (rand > 5) {
+                rand = 5;
+            }
+
+            mRotate[idx] = rot[rand];
+        }
+
+        letter[idx].mUserArea += 1;
+
+        f32 y = letter[idx].mUserArea / 7.0f;
+        f32 y2 = (f32)letter[idx].mUserArea * (f32)letter[idx].mUserArea / 49.0f;
+        setRotate(&letter[idx], (f32)mRotate[idx] * (1.0f - y));
+        fopMsgM_paneTrans(&letter[idx], 0.0f, y2 * 244.0f);
+    }
+    
+    if (letter[idx].mUserArea == 7) {
+        fopMsgM_setNowAlphaZero(&letter[idx]);
+        ret = TRUE;
+    }
+
+    return ret;
 }
 
 /* 8018F4A8-8018F548       .text setRotate__24dDlst_GameOverScrnDraw_cFP18fopMsgM_pane_classf */
 void dDlst_GameOverScrnDraw_c::setRotate(fopMsgM_pane_class* pane, f32 angle) {
-    /* Nonmatching */
+    // Fakematch?
+    f32 y = (s32)(pane->mSize.y / 2);
     J2DPane* j2dpane = pane->pane;
-    j2dpane->mBasePosition.set((s32)pane->mSize.x / 2.0f, (s32)pane->mSize.y / 2.0f);
+    j2dpane->mBasePosition.set((s32)(pane->mSize.x / 2), y);
```

Comment:
this is probably `J2DPane::rotate(f32, f32, J2DRotateAxis, f32)`. the inline is probably supposed to be implemented along these lines:
```cpp
mBasePosition.x = offsetX;
mBasePosition.y = offsetY;
mRotationAxis = axis;
mRotation = angle;
calcMtx();
```